Autopilot/Inc/AttitudePathInterface.hpp

```diff
@@ -36,15 +36,6 @@ extern "C"
 
 const char PATH_ATTITUDE_MAIL_Q_SIZE = 1;
 
-//Set up a mail queue for sending commands to the attitude manager
-extern osMailQDef(commandsMailQ, PATH_ATTITUDE_MAIL_Q_SIZE, CommandsForAM);
```

Comment:
I'm confused as to why your'e getting rid of this. How will each thread know about the other queue ?

Reply:
These definitions still exist but have been moved to the cpp files to avoid linking issues.